_includes/sections/instant-messenger.html

```diff
@@ -1,83 +1,268 @@
+    </ul>
+
+    <h5>October 2019</h5>
+    <ul>
+        <li><a href="https://www.eff.org/deeplinks/2019/10/open-letter-governments-us-uk-and-australia-facebook-all-out-attack-encryption">The Open Letter from the Governments of US, UK, and Australia to Facebook is An All-Out Attack on Encryption (EFF)</a></li>
+        <li><a href="https://arstechnica.com/tech-policy/2019/10/the-broken-record-why-barrs-call-against-end-to-end-encryption-is-nuts/">The broken record: Why Barr’s call against end-to-end encryption is nuts (ArsTechnica)</a></li>
+        <li><a href="https://arstechnica.com/information-technology/2019/10/ag-barr-is-pushing-facebook-to-backdoor-whatsapp-and-halt-encryption-plans">US wants Facebook to backdoor WhatsApp and halt encryption plans (ArsTechnica)</a></li>
+    </ul>
+
+    <h5>August 2019</h5>
+    <ul>
+        <li><a href="https://arstechnica.com/tech-policy/2019/08/post-snowden-tech-became-more-secure-but-is-govt-really-at-risk-of-going-dark">Post Snowden tech became more secure, but is government really at risk of going dark? (ArsTechnica)</a></li>
+    </ul>
+
+    <h5>July 2019</h5>
+    <ul>
+        <li><a href="https://techcrunch.com/2019/07/23/william-barr-consumers-security-risks-backdoors/">US attorney general William Barr says Americans should accept security risks of encryption backdoors (TechCrunch)</a></li>
+        <li><a href="https://www.theregister.co.uk/2019/07/23/us_encryption_backdoor/">Low Barr: Don't give me that crap about security, just put the backdoors in the encryption, roars US Attorney General (The Register)</a></li>
+    </ul>
+
+    <h5>May 2019</h5>
+    <ul>
+        <li><a href="https://www.theguardian.com/uk-news/2019/may/30/apple-and-whatsapp-condemn-gchq-plans-to-eavesdrop-on-encrypted-chats">Apple and WhatsApp condemn GCHQ plans to eavesdrop on encrypted chats (The Guardian)</a></li>
+    </ul>
+
+    <h5>January 2019</h5>
+    <ul>
+        <li><a href="https://www.justsecurity.org/62114/give-ghost-backdoor/">Give Up the Ghost: A Backdoor by Another Name (Just Security)</a></li>
+    </ul>
+
+    <h5>December 2018</h5>
+    <ul>
+        <li><a href="https://www.zdnet.com/article/whats-actually-in-australias-encryption-laws-everything-you-need-to-know/">What's actually in Australia's encryption laws? Everything you need to know (ZDnet)</a></li>
+    </ul>
+
+    <h3>Complete Comparison</h3>
+    <ul>
+        <li><a href="https://securechatguide.org/effguide.html">securechatguide.org</a> - Guide to Choosing a Messenger.</li>
+        <li><a href="https://www.securemessagingapps.com/">securemessagingapps.com</a> - Secure Messaging Apps Comparison.</li>
+        <li><a href="https://www.thinkprivacy.io/messengers.html">thinkprivacy.io</a> - Simple Secure Messaging Apps Comparison.</li>
+    </ul>
+
+    <h3>Independent security audits</h3>
```

Comment:
I'm thinking for Signal and Keybase, we could just link to this subsection instead (since it already contains links to the audits and more)?
```suggestion
    <h3 id="#rtc-independent-security-audits">Independent security audits</h3>
```